lib/src/layout/scaffold.dart

```diff
@@ -1,118 +1,331 @@
-import 'package:macos_ui/src/library.dart';
+import 'dart:math' as math;
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
 import 'package:macos_ui/macos_ui.dart';
-import 'package:split_view/split_view.dart';
+import 'package:macos_ui/src/layout/content_area.dart';
+import 'package:macos_ui/src/layout/resizable_pane.dart';
+import 'package:macos_ui/src/layout/resizable_pane_notifier.dart';
+import 'package:macos_ui/src/layout/sidebar.dart';
+import 'package:macos_ui/src/layout/title_bar.dart';
+
+const _kTitleBarHeight = 52.0;
```

Comment:
Add docs

Reply:
I don't think it's necessary to add docs to this one because its private.

Reply:
Fair

Reply:
Is there a particular reason this variable is private?

The AppBar for example uses `kToolbarHeight` which is public available.
https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/constants.dart#L22

I think it might come in handy to get access to these kinds of parameters as a developer.